app/src/main/res/values/strings.xml

```diff
@@ -303,7 +303,7 @@
 
   <string name="about_translate_cancel">Cancel</string>
   <string name="retry">Retry</string>
-  <string name="showcase_view_whole_nearby_activity">These are the places near you that need pictures to illustrate their Wikipedia articles.\n\nClicking on \'SEARCH THIS AREA\' locks the map and launches a nearby search around that location.</string>
+  <string name="showcase_view_whole_nearby_activity">These are the places near you that need pictures to illustrate their Wikipedia articles.\n\nClicking on \'Search this area\' locks the map and launches a nearby search around that location.</string>
```

Comment:
There is no "Search this area" button anymore.
Actually, is this string used somewhere? If you find where it is used, would you mind posting a screenshot?
Thanks a lot! 🙂

Reply:
I found that it was used in [dialog_nearby.xml](https://github.com/commons-app/apps-android-commons/blob/main/app/src/main/res/layout/dialog_nearby.xml#L15), but I believe the activity using the layout has been refactored.

Reply:
Would you mind trying to remove it? Any impact? 🙂

Reply:
Deleting the button and its string seems fine, but the rest of the content in [dialog_nearby](https://github.com/commons-app/apps-android-commons/blob/main/app/src/main/res/layout/dialog_nearby.xml) are still used when providing notifications in the map explore section.

Reply:
Hey Nicholas after further review I noticed that the [dialog_nearby.xml](https://github.com/commons-app/apps-android-commons/blob/main/app/src/main/res/layout/dialog_nearby.xml) file had no usages in the whole app. I have safe deleted the file and its associated string resources and it appears to build fine.